server/routerlicious/packages/services/src/documentManager.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ScopeType } from "@fluidframework/protocol-definitions";
+import { BasicRestWrapper } from "@fluidframework/server-services-client";
+import { IDocumentManager, IDocument } from "@fluidframework/server-services-core";
+import { generateToken, getCorrelationId } from "@fluidframework/server-services-utils";
+
+/**
+ * Manager to fetch document from Alfred using the internal URL.
+ */
+export class DocumentManager implements IDocumentManager {
+
+	constructor(
+		private readonly internalAlfredUrl: string,
+		private readonly key: string
```

Comment:
@arikt-ms , I talked to you earlier that I should move `tenantId` to this constructor. But I feel without it may support us to authentic using client client credentials auth flow. So will probably leave it this way. Let me know if you have any concerns